### PR TITLE
[3.8] CEQ-10309 Incorrect links in product.json

### PR DESCRIPTION
**Pull request by ppalaga**
https://issues.redhat.com/browse/CEQ-10309

### CODE REVIEW

File: product/src/main/resources/camel-quarkus-product-source.json [hunk 1]
```diff
@@ -617,7 +617,7 @@
         "quarkus-community.version": "${version?replace('\\\\.redhat-.*$', '', 'r')}",
         "smallrye.reactive.messaging.camel.version": "${version?replace('\\\\.redhat-.*$', '', 'r')}"
     },
-    "guideUrlTemplate": "https://access.redhat.com/documentation/en-us/red_hat_build_of_apache_camel/${productMajorMinorVersion}/html/red_hat_build_of_apache_camel_for_quarkus_reference/camel-quarkus-extensions-reference#extensions-${artifactIdBase}",
+    "guideUrlTemplate": "https://content-stage.docs.redhat.com/en/documentation/red_hat_build_of_apache_camel/${productMajorMinorVersion}/html-single/red_hat_build_of_apache_camel_for_quarkus_reference/camel-quarkus-extensions-reference#extensions-${artifactIdBase}",
```
Comment (rhaetor) [inline product/src/main/resources/camel-quarkus-product-source.json]:
```suggestion
    "guideUrlTemplate": "https://docs.redhat.com/en/documentation/red_hat_build_of_apache_camel/${productMajorMinorVersion}/html-single/red_hat_build_of_apache_camel_for_quarkus_reference/camel-quarkus-extensions-reference#extensions-${artifactIdBase}",
```

File: product/src/main/resources/quarkus-cxf-product-source.json [hunk 2]
```diff
@@ -42,5 +42,5 @@
             "comment": "supported/supported since CEQ 2.13.3/QCXF 1.5"
         }
     },
-    "guideUrlTemplate": "https://access.redhat.com/documentation/en-us/red_hat_build_of_apache_camel/${productMajorMinorVersion}/html/red_hat_build_of_quarkus_cxf_reference/quarkus-cxf-extensions-reference#extensions-${artifactId}"
+    "guideUrlTemplate": "https://content-stage.docs.redhat.com/en/documentation/red_hat_build_of_apache_camel/${productMajorMinorVersion}/html-single/red_hat_build_of_apache_camel_for_quarkus_reference/camel-quarkus-extensions-reference#${artifactId}"
```
Comment (rhaetor) [inline product/src/main/resources/quarkus-cxf-product-source.json]:
```suggestion
    "guideUrlTemplate": "https://docs.redhat.com/en/documentation/red_hat_build_of_apache_camel/${productMajorMinorVersion}/html-single/red_hat_build_of_apache_camel_for_quarkus_reference/camel-quarkus-extensions-reference#${artifactId}"
```

**Comment by ppalaga** [comment]
Thanks @rhaetor, fixed in 05ec584. Could you please re-review?

**Comment by rhaetor** [comment]
@ppalaga Looks good now, thanks.

**Pull request merged**